backend/src/main/java/com/redhat/cloud/notifications/db/repositories/SubscriptionRepository.java

```diff
@@ -40,33 +43,52 @@ public class SubscriptionRepository {
     @Inject
     TemplateRepository templateRepository;
 
+    private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Any reason why you didn't inject the `ObjectMapper` bean provided by Quarkus?

Reply:
no reason indeed, fixed